enterprise/reporting/reporting.go

```diff
@@ -70,14 +72,15 @@ type HandleT struct {
 }
 
 func NewFromEnvConfig(log logger.Logger) *HandleT {
-	var sleepInterval, mainLoopSleepInterval time.Duration
+	var sleepInterval, mainLoopSleepInterval, dbQueryTimeout time.Duration
 	var maxOpenConnections int
 	reportingServiceURL := config.GetString("REPORTING_URL", "https://reporting.rudderstack.com/")
 	reportingServiceURL = strings.TrimSuffix(reportingServiceURL, "/")
 	sourcesWithEventNameTrackingDisabled := config.GetStringSlice("Reporting.sourcesWithEventNameTrackingDisabled", []string{})
 
 	config.RegisterDurationConfigVariable(5, &mainLoopSleepInterval, true, time.Second, "Reporting.mainLoopSleepInterval")
 	config.RegisterDurationConfigVariable(30, &sleepInterval, true, time.Second, "Reporting.sleepInterval")
+	config.RegisterDurationConfigVariable(60, &dbQueryTimeout, true, time.Second, "Reporting.dbQueryTimeout")
```

Comment:
Please switch to the new reloadable config api for all configuration options, e.g.

```go
var dbQueryTimeout *config.Reloadable[time.Duration]


dbQueryTimeout = config.GetReloadableDurationVar(60, time.Second, "Reporting.dbQueryTimeout")


```